src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -17,6 +17,11 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.7'">
     <TestNugetTargetMoniker Include="$(NugetTargetMoniker)">
+      <Folder>ns1.7-netcoreapp1.1</Folder>
+    </TestNugetTargetMoniker>
+  </ItemGroup>
```

Comment:
This should probably go in dir.props as I suppose a lot of test projects will start doing this.